<launch>

  <arg name="tf_prefix" default="" />

  <arg name="out_topic" default="scan_filtered" />
  <arg name="min_height" default="-0.3" />
  <arg name="max_height" default="0.3" />

  <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan">
    <param name="min_height" value="$(arg min_height)" />
    <param name="max_height" value="$(arg max_height)" />
    <param name="angle_min" value="-3.1416" /> <!-- -pi -->
    <param name="angle_max" value="3.1416" /> <!-- pi -->
    <param name="scan_time" value="0.1" />
    <param name="range_max" value="40.0" />
    <param name="range_min" value="0.4" />
    <remap from="cloud_in" to="velodyne_points"/>
    <remap from="scan" to="velodyne/scan" />
    <param name="use_inf" value="true" />
  </node>

  <include file="$(find segbot_sensors)/launch/velodyne/velodyne-filters.launch">
    <arg name="in_topic" value="velodyne/scan" />
    <arg name="out_topic" value="velodyne/$(arg out_topic)" />
  </include>

</launch>

